src/mumble/UserView.cpp

```diff
@@ -198,7 +198,7 @@ void UserView::mouseReleaseEvent(QMouseEvent *evt) {
 		} else if (channel && !channel->qbaDescHash.isEmpty()) {
 			hasComment = true;
 
-			if (channel->bFiltered)
+			if (channel->m_filterMode != Channel::FILTER_MODE_NORMAL)
 				commentIconPxOffset -= m_iconTotalDimension;
```

Comment:
For future-proofing this code, a switch-statement (with fallthrough) might be better here.